sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -288,20 +291,69 @@ public static void trackEvent(String name, Map<String, String> properties) {
     }
 
     /**
-     * Track a custom event with name and optional properties and optional transmissionTarget.
+     * Track a custom event with name and optional properties.
      * The name parameter can not be null or empty. Maximum allowed length = 256.
      * The properties parameter maximum item count = 20.
      * The properties keys can not be null or empty, maximum allowed key length = 125.
-     * The properties values can not be null, maximum allowed value length = 125.
+     * The property values cannot be null.
+     * The property string maximum allowed length = 125.
```

Comment:
:warning: Validation missed, but tests passed

Reply:
Yes the validation was planned to be added in AnalyticsValidator for AppCenter as One Collector allows empty strings. As for the Javadoc we will need to do something now that we have One Collector having different rules.